### PR TITLE
04 get /api/articles

### PR DESCRIPTION
**Pull request by siandaniel**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -62,3 +62,77 @@ describe("/api/topics", () => {
     });
 });
 
+describe("/api/articles", () => {
+    describe("GET", () => {
+        test("Returns 'Status: 200' if no error in path", () => {
+            return request(app).get('/api/articles').expect(200);
+        });
```
Comment (NRMSMITH) [inline __tests__/app.test.js:69]: This test is unnecessary, as you will be expecting '200' for the status on the next test :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -62,3 +62,77 @@ describe("/api/topics", () => {
...
+                expect(articles[0]).toHaveProperty("topic", 'mitch');
+                expect(articles[0]).toHaveProperty("created_at", '2020-11-03T09:12:00.000Z');
+                expect(articles[0]).toHaveProperty("votes", 0);
+                expect(articles[0]).toHaveProperty("article_img_url", 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700');
+                expect(articles[0]).toHaveProperty("comment_count", '2');
+            });
+        });
+        test("Returns 'Status: 404' if invalid path provided to articles, e.g. misspelt path", () => {
+            return request(app).get('/api/articlees').expect(404);
+        });
```
Comment (NRMSMITH) [inline __tests__/app.test.js:130]: Same as above ^^

**Review by NRMSMITH — APPROVED** [review]
Great work Sian! 👍  I've added comments about testing that you don't need, but you can move on 😄

File: __tests__/app.test.js (hunk 1 above)
Comment (siandaniel) [reply __tests__/app.test.js:69]: Thanks, Niamh! :)

**Pull request merged**